package.json

```diff
@@ -10,59 +10,53 @@
     "browser-test-remote-ie": "nightwatch --config nightwatch.js --env ie",
     "browser-test-remote-safari": "nightwatch --config nightwatch.js --env safari",
     "browser-test-remote-parallel": "nightwatch --config nightwatch.js --env safari,ie,default,chrome",
-    "build": "mkdir build; browserify src/index.js -g yo-yoify -o build/app.js -t [ babelify ]; babel --plugins=transform-es2015-template-literals build/app.js --out-file build/app.js",
+    "build": "mkdirp build; browserify src/index.js -t yo-yoify -t [ babelify --presets [ es2015 ] ] | babili > build/app.js",
     "csslint": "csslint --ignore=order-alphabetical --errors='errors,duplicate-properties,empty-rules' --exclude-list='assets/css/font-awesome.min.css' assets/css/",
-    "downloadsolc": "rm soljson.js; wget https://ethereum.github.io/solc-bin/soljson.js",
-    "lint": "standard",
+    "downloadsolc": "rm -f soljson.js; wget https://ethereum.github.io/solc-bin/soljson.js",
+    "lint": "standard | notify-error",
+    "make-mock-compiler": "node ci/makeMockCompiler.js",
+    "onchange": "onchange build/app.js -- npm run lint",
+    "prepublish": "npm-run-all -ls downloadsolc make-mock-compiler build",
+    "selenium": "selenium-standalone start",
     "serve": "http-server .",
-    "test": "standard; babel src --out-dir babelify-src; node test/index.js"
+    "start": "npm-run-all -lpr serve watch onchange",
+    "test": "standard; babel src --out-dir babelify-src; node test/index.js",
+    "watch": "mkdirp build; watchify src/index.js -p [ browserify-livereload --host 127.0.0.1 --port 1337 ] -t [ babelify --presets [ es2015 ] --no-sourceMaps ] -dv -o build/app.js"
```

Comment:
same as above. it's a command line interfcace tool that is cross platform and can be installed via npm, so `npm install` will actually install `mkdirp` so that it will be available cross platform